### PR TITLE
kRO patch 2022-02-03/04 items

### PR DESCRIPTION
**Pull request by Toshiro90**
* **Addressed Issue(s)**:  -

* **Server Mode**: Renewal

* **Description of Pull Request**: 

Adds new items and updated some existing ones, based on kRO patches from 2022-02-03/04. Includes 3 new items from kRO Zero patch.

### CODE REVIEW

File: db/re/item_db_usable.yml [hunk 1]
```diff
@@ -49878,6 +49892,103 @@ Body:
...
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* synthesisui 101311; */
+  - Id: 101314
+    AegisName: aegis_101314
+    Name: Helm of Faith Box
+    Type: Etc
```
Comment (Takerio) [inline db/re/item_db_usable.yml:49965]: wrong type^^ its Type: Usable

File: db/re/item_db_etc.yml [hunk 2]
```diff
@@ -62381,6 +62538,12 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+  - Id: 1000854
+    AegisName: aegis_1000854
```
Comment (Everade) [inline db/re/item_db_etc.yml:62542]: AegisName: Shadow_Exchange_Ticket

File: db/re/item_db_usable.yml [hunk 3]
```diff
@@ -49878,6 +49892,103 @@ Body:
       NoAuction: true
     Script: |
       percentheal 30,30;
+  - Id: 101279
+    AegisName: 2022_LunarNewYears_box
```
Comment (Everade) [inline db/re/item_db_usable.yml:49896]: Maybe add note for missing event item script.

**Review by Everade — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -139998,7 +140017,6 @@ Body:
     Trade:
       NoDrop: true
```
[the remark this answers is not in the document]
Comment (Toshiro90) [reply db/re/item_db_equip.yml:140018]: Trade restrictions of newly added (or updated) items are based on entries in data/ItemMoveInfoV5.txt, but it seems GitHub is unable to properly display the change in it's web preview.
The item with updated trade restrictions is 20616/C_School_Bag_PU and not 440000/C_SharkHead (which already has no restrictions) as previewed.

[the remark this answers is not in the document]
Comment (Everade) [reply db/re/item_db_equip.yml:140018]: You are right. What a pity.

File: db/re/item_db_usable.yml [hunk 5]
```diff
@@ -49878,6 +50012,105 @@ Body:
...
+      NoSell: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 1000274,10;
+      getitem 100081,3;
+      getitem 25464,20;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:50113]: getgroupitem?

File: db/re/item_db_usable.yml [hunk 6]
```diff
@@ -50094,6 +50228,103 @@ Body:
...
+      /* reformui 101307; */
+  - Id: 101308
+    AegisName: True_GemShadow_Up
+    Name: True Gemstone Shadow Spellbook
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* synthesisui 101308; */
```
Comment (Atemo) [inline db/re/item_db_usable.yml:50272]:
```suggestion
      laphine_synthesis();
```

File: db/re/item_db_usable.yml [hunk 7]
```diff
@@ -50094,6 +50228,103 @@ Body:
...
+  - Id: 101309
+    AegisName: EXPShadow_Up
+    Name: Experience Shadow Spellbook
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* synthesisui 101309; */
```
Comment (Atemo) [inline db/re/item_db_usable.yml:50281]:
```suggestion
      laphine_synthesis();
```

File: db/re/item_db_usable.yml [hunk 8]
```diff
@@ -50094,6 +50228,103 @@ Body:
...
+  - Id: 101310
+    AegisName: MagicalShadow_Up
+    Name: Clever Shadow Spellbook
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* synthesisui 101310; */
```
Comment (Atemo) [inline db/re/item_db_usable.yml:50290]:
```suggestion
      laphine_synthesis();
```

File: db/re/item_db_usable.yml [hunk 9]
```diff
@@ -50094,6 +50228,103 @@ Body:
...
+  - Id: 101311
+    AegisName: InfinityShadow_Up
+    Name: Infinity Shadow Spellbook
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* synthesisui 101311; */
```
Comment (Atemo) [inline db/re/item_db_usable.yml:50299]:
```suggestion
      laphine_synthesis();
```

**Review by Atemo — APPROVED** [review]

File: db/re/item_db_usable.yml (hunk 6 above)
Comment (Everade) [reply db/re/item_db_usable.yml:50272]: laphine_synthesis db entry is missing.

File: db/re/item_db_usable.yml (hunk 7 above)
Comment (Everade) [reply db/re/item_db_usable.yml:50281]: laphine_synthesis db entry is missing.

File: db/re/item_db_usable.yml (hunk 8 above)
Comment (Everade) [reply db/re/item_db_usable.yml:50290]: laphine_synthesis db entry is missing.

File: db/re/item_db_usable.yml (hunk 9 above)
Comment (Everade) [reply db/re/item_db_usable.yml:50299]: laphine_synthesis db entry is missing.

File: db/re/item_db_usable.yml (hunk 6 above)
Comment (Atemo) [reply db/re/item_db_usable.yml:50272]: It's just one more.. many items have the laphine_synthesis command without the db (`3Lv_9Refine_Weapon_7Gu`...) from the laphine PR

**Pull request merged**